srcpkgs/wayfire/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'wayfire'
 pkgname=wayfire
-version=0.7.5
-revision=2
-_utils_commit=889fdafa7b09fc6ce70313fd3f067cfc8c21a2ac
+version=0.8.1
+revision=1
+_utils_commit=08553c418f164bf5e84613d27447a32e380b75f0
```

Comment:
This commit hash is incorrect; it points to HEAD, not to the version wayfire 0.8.1 expects. If you have questions, I've created a mini-tutorial explaining exactly this problem: https://xbps-src-tutorials.github.io/mini-tutors/git-submodules.html It explains it from a template creation standpoint, but most of it also applies here when updating templates.

```suggestion
_utils_commit=15f8e16721585ae3eaf278ba71d7064237eb23f5
```

Reply:
neat, thank you !
Saving this one for the future !